components/Fonts.tsx

```diff
@@ -24,8 +24,6 @@ const nunito = Nunito({
 export default function Fonts() {
   return (
     <style
-      jsx={true}
-      global={true}
```

Comment:
There were warnings about these invalid attributes so I removed them. I don't noticed anything that breaks by removing them, but let me know if they are supposed to be there for some reason.